### DISCUSSION TITLE
Carbon Intensity API from developer in Japan

### DISCUSSION THREAD

**Discussion by jacmacmod** [electricitymaps/electricitymaps-contrib#2821] [Ideas]
Hello my acquaintance in [Japan](https://denkicarbon.jp/) is working on a[ Carbon intensity API](https://github.com/FraserTooth/japan_grid_carbon_api) for all of Japan. Perhaps his API or data will help in filling out the real time Japan data. I was also wondering what is the current method of finding real time data for Japan?

**Comment by FelixDQ** [comment]
That looks interesting. The values that we are interested in is the power production breakdown.

Our current data sources for japan can be found here: https://github.com/tmrowco/electricitymap-contrib/blob/master/DATA_SOURCES.md#real-time-electricity-data-sources
